tests/unit/test_fhir_mcp_server.py

```
@@ -74,6 +75,22 @@ async def test_get_token_cached_within_ttl() -> None:
+async def test_refresh_token_invalid_expires_in_falls_back_to_default() -> None:
+    server = _make_server()
+    ok = _http_response(
+        200,
+        json_data={"access_token": "tok-x", "expires_in": "not-a-number"},
+        url=server._token_url,
+    )
+    mock_http = AsyncMock()
+    mock_http.post = AsyncMock(return_value=ok)
+    server._http = mock_http
+    await server._refresh_token()
+    assert server._access_token == "tok-x"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "_access_token"

(S105)